Rendering/include/Resources/Primitives.h

```diff
@@ -41,45 +39,6 @@ struct Shape
     std::vector<uint32_t> indices;
 };
 
-// Derived struct for Triangle
-struct Triangle : public Shape
-{
-    Triangle()
-    {
-        vertices.resize(3);
-        vertices.resize(3);
-        vertices =
-        {
-            Vertex(glm::vec3(-0.5f, -0.5f, 0.0f)),  // 0
-            Vertex(glm::vec3( 0.0f,  0.5f, 0.0f)),  // 1
-            Vertex(glm::vec3( 0.5f, -0.5f, 0.0f))   // 2
-        };
-        indices = { 0, 1, 2 };
-    }
-};
-
-// Derived struct for Square
-struct Square : public Shape
-{
-    Square()
-    {
-        vertices.resize(4);
-        vertices.resize(6);
-        vertices =
-        {
-            Vertex(glm::vec3(-0.5f, -0.5f, 0.0f)), // 0
-            Vertex(glm::vec3( 0.5f, -0.5f, 0.0f)), // 1
-            Vertex(glm::vec3( 0.5f,  0.5f, 0.0f)), // 2
-            Vertex(glm::vec3(-0.5f,  0.5f, 0.0f))  // 3
-        };
-        indices =
-        {
-            0, 1, 2,
-            2, 3, 0
-        };
-    }
-};
-
```

Comment:
Removed becasue it's 2D?

Reply:
Bloat and unlikely to be used as is. Maybe as billboards but we'd have a billboard class.